src/btree/bt_ovfl.c

```diff
@@ -59,8 +59,9 @@ __wt_ovfl_read(WT_SESSION_IMPL *session, WT_PAGE *page, WT_CELL_UNPACK_COMMON *u
     /*
      * WT_CELL_VALUE_OVFL_RM cells: if reconciliation deletes an overflow value, the on-page cell
      * type is reset to WT_CELL_VALUE_OVFL_RM. Any values required by an existing reader will be
-     * copied into the HS file, which means this value should never be read. It's possible to race
-     * with checkpoints doing that work, lock before testing the removed flag.
+     * copied into the HS file, which means this value should never be read. Return not found for
+     * the upper layer to retry the read. It's possible to race with checkpoints doing that work,
+     * lock before testing the removed flag.
```

Comment:
Where is the lock before testing the removed flag in the code changes?

Reply:
I reverted the code change should revert the comment change as well.